### PR TITLE
Debian packaging updates

### PR DESCRIPTION
**Pull request by sts**
- Update systemd service to match the renamed parameter
- Update example configs
- Update Github workflow

### CODE REVIEW

File: .github/workflows/package.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
   push:
     branches:
       - main
-      - test
+      - sts_fix_debian_package
```
Comment (zc-devs) [inline .github/workflows/package.yaml:8]: Why this branch?

File: .github/workflows/package.yaml [hunk 2]
```diff
@@ -46,7 +46,7 @@ jobs:
           sudo apt update && sudo apt -y install make pkg-config rubygems && sudo gem install fpm
 
           # Download corazawaf/coraza/coraza.conf for distribution
-          curl https://raw.githubusercontent.com/corazawaf/coraza/v2/master/coraza.conf-recommended > coraza.conf
+          curl https://raw.githubusercontent.com/corazawaf/coraza/v3.0.2/coraza.conf-recommended > coraza.conf
```
Comment (zc-devs) [inline .github/workflows/package.yaml:49]: Currently we are using [3.0.0](https://github.com/corazawaf/coraza-spoa/blob/dd5eb86d1e9abbdd5fe568249f36a6d85257eba7/go.mod#L7). So, let's make version bump synchronous in another PR.

File: contrib/coraza-spoa.service [hunk 3]
```diff
@@ -3,14 +3,14 @@ Description=Coraza WAF SPOA Daemon
...
-ExecStart=/usr/bin/coraza-spoa -config-file=/etc/coraza-spoa/config.yaml
+ExecStart=/usr/bin/coraza-spoa -config=/etc/coraza-spoa/config.yaml
 WorkingDirectory=/
 Restart=always
 Type=exec
 User=coraza-spoa
 Group=coraza-spoa
 
-# Hardening
+## Hardening
```
Comment (zc-devs) [inline contrib/coraza-spoa.service:13]: Why double hash?

**Comment by jcchavezs** [comment]
Ping @sts

**Comment by jcchavezs** [comment]
@sts I just noticed this PR was merged without passing the build stage, any comment on this @sts ?
